corehq/apps/data_interfaces/views.py

```diff
@@ -155,6 +155,69 @@ def get_deleted_item_data(self, item_id):
         }
 
 
+class ArchiveFormView(DataInterfaceSection):
```

Comment:
ah you're right. haha good catch. i just looked again at the test and the user i created to test that it needed to be superuser, didn't have membership to the domain and that's why it was passing